### PR TITLE
Set up llvm for osmesa.

### LINKED ISSUE #18954: TravisCI is broken since osmesa update
https://gist.github.com/jdm/f059bf33a5fa3c5a089771806e1dbaba

### PR DESCRIPTION
**Pull request by jdm**
This fixes #18954.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/18955)

### CODE REVIEW

**Comment by Nashenas88** [comment]
@jdm will this require a change to the build instructions in the README.md as well? I just did a test build on a new ubuntu machine and it broke.

**Comment by emilio** [comment]
@Nashenas88 what does the failure look like? Can you file an issue?

@bors-servo p=1

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,12 @@ matrix:
       cache: false
     - sudo: 9000
       dist: trusty
+      before_install:
+        - sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+        - sudo add-apt-repository 'deb http://apt.llvm.org/precise/ llvm-toolchain-precise-3.9 main' -y
```
Comment (SimonSapin) [inline .travis.yml]: This should likely be `trusty` instead of `precise`, since we have `dist: trusty` in the same file.

**Pull request merged**